geode-core/src/main/java/org/apache/geode/internal/tcp/Connection.java

```diff
@@ -3047,7 +3042,8 @@ private boolean readMessageHeader(ByteBuffer peerDataBuffer) throws IOException
     return false;
   }
 
-  private void readMessage(ByteBuffer peerDataBuffer, AbstractExecutor threadMonitorExecutor) {
+  @VisibleForTesting
```

Comment:
Package private methods don't really need this annotation. Making methods package private is normal for unit testing. This annotation should be reserved for public methods.

Reply:
Fixed